README.md

```diff
@@ -141,7 +163,9 @@ By setting `LOG_STRIPE_SERVICE` to true in your application configuration you ca
 ````javascript
 var ENV = {
   // some vars...
-  LOG_STRIPE_SERVICE: true,
+  stripe: {
```

Comment:
this variable perhaps should be renamed, it implies that you are configuring Stripe as a whole, and not the service we are providing, which is why the above `LOG_STRIPE_SERVICE` was so explicit. This particular item is a -1 for me.

you could solve it by making the `debug` variable a bit more explicit to indicate you are debugging the service and not stripe's library, or you could rename the `stripe` scope into something that is specific to the ember service. I'm overall not a fan of the nesting and change, but as long as the intent of this variable is clearly defined in the code, w/o any additional documentation context then I can +1

Reply:
`stripe-service` ok?

Reply:
yes, i think that should work. thanks!

Reply:
👍

Reply:
I'm taking a crack at this today. It feels a little odd to have config split across ENV['stripe'] and ENV['stripe-service']. We should combine to one place for easy of use. I think ENV['stripe'] makes the most sense.